packages/components/Menu/src/MenuList/MenuList.tsx

```diff
@@ -83,7 +83,7 @@ export const MenuList = compose<MenuListType>({
 
       const content =
         Platform.OS === 'macos' ? (
-          <Slots.root>
+          <Slots.root onMouseLeave={() => focusZoneRef.current.focus()}>
```

Comment:
Might be good to extract this to a a separate function / React.useCallback because we try to not pass lamba functions directly in jsx.